CHANGELOG.md

```diff
@@ -1,253 +1,274 @@
-# Change log
+<!-- markdownlint-disable MD024 -->
+# Changelog
 
-All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
+All notable changes to this project will be documented in this file.
```

Comment:
was the changelog rewrite intentional?

Reply:
Yes this is auto-generated by our new changelog generator 💯